src/couch_multidb_changes.erl

```diff
@@ -145,7 +145,7 @@ handle_cast({resume_scan, DbName}, #state{pids=Pids, tid=Ets} = State) ->
 
 
 handle_info({'EXIT', From, normal}, #state{scanner = From} = State) ->
-    couch_log:info("multidb_changes ~p scanner pid exited ~p",[State#state.suffix, From]),
+    couch_log:debug("multidb_changes ~p scanner pid exited ~p",[State#state.suffix, From]),
     {noreply, State#state{scanner=nil}};
```

Comment:
In this case it is a normal exit and we don't restart scanner. It is a debug message saying "scanner has finished finding all local db shards and exited". From then on event handler on local db shards will handle updates / new db shards.